client/lib/site-roles/test/store.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'Viewers Store', () => {
+	let Dispatcher, RolesStore;
+	let actions, site, siteId;
+
+	useMockery();
+	useFakeDom();
```

Comment:
Can we double check if we really need `useFakeDom` in this test. I noticed it was overused in many places. It seems like copy/paste artefact :)